server/tests/snapshots/snap_test_multiple_targeted_contests.py

```diff
@@ -41,3 +41,870 @@
     "Contest 3 - candidate 2": 0,
     "Contest 3 - candidate 3": 0,
 }
+
+snapshots[
+    "test_two_rounds 4"
+] = """######## ELECTION INFO ########\r
+Election Name,State\r
+Test Election,CA\r
+\r
+######## CONTESTS ########\r
+Contest Name,Targeted?,Number of Winners,Votes Allowed,Total Ballots Cast,Tabulated Votes\r
+Contest 1,Targeted,1,1,1600,candidate 1: 600; candidate 2: 400\r
+Contest 2,Targeted,1,1,1600,Yes: 800; No: 650\r
+Contest 3,Opportunistic,2,2,600,candidate 1: 200; candidate 2: 300; candidate 3: 100\r
+\r
+######## AUDIT SETTINGS ########\r
+Audit Name,Risk Limit,Random Seed,Online Data Entry?\r
+Test Audit,10%,1234567890,Yes\r
+\r
+######## AUDIT BOARDS ########\r
+Jurisdiction Name,Audit Board Name,Member 1 Name,Member 1 Affiliation,Member 2 Name,Member 2 Affiliation\r
+\r
+######## ROUNDS ########\r
+Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
+1,Contest 1,Targeted,191,Yes,1.84402281147622e-05,DATETIME,DATETIME,candidate 1: 132; candidate 2: 59\r
+1,Contest 2,Targeted,485,No,1.0,DATETIME,DATETIME,Yes: 0; No: 0\r
+1,Contest 3,Opportunistic,,No,1.0,DATETIME,DATETIME,candidate 1: 0; candidate 2: 0; candidate 3: 0\r
+2,Contest 2,Targeted,1468,Yes,8.27580207807509e-24,DATETIME,DATETIME,Yes: 1028; No: 440\r
+2,Contest 3,Opportunistic,,No,1.0,DATETIME,DATETIME,candidate 1: 0; candidate 2: 0; candidate 3: 0\r
+\r
+######## SAMPLED BALLOTS ########\r
+Jurisdiction Name,Batch Name,Ballot Position,Audited?,Ticket Numbers: Contest 1,Ticket Numbers: Contest 2,Audit Result: Contest 1,Audit Result: Contest 2,Audit Result: Contest 3\r
+J1,1,4,AUDITED,,"Round 1: 0.324042377098893634, Round 2: 0.781169515578102262",candidate 1,Yes,\r
+J1,1,9,AUDITED,Round 1: 0.132689946437387284,Round 1: 0.132689946437387284,candidate 1,,\r
```

Comment:
@umbernhard would we expect a ballot that got sampled once in each of the targeted contests to have the same ticket number for each contest like we see here? The contests have the same universe but diff margins (and thus diff sample sizes). Also note similar behavior on line 77 below.

Reply:
Is that right? Are we using the same seed for each contest, no contest index involved?